src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/VtableIndexStubGenerator.cs

```diff
@@ -11,28 +11,29 @@
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.Interop;
 using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
 
 [assembly: System.Resources.NeutralResourcesLanguage("en-US")]
 
+internal sealed record IncrementalMethodStubGenerationContext(
```

Comment:
Do we want this in the global namespace, or in Microsoft.Interop?

Reply:
I'll move this back into the namespace. I've gotten too used to file-scoped namespaces.